core/capabilities/remote/trigger_subscriber.go

```diff
@@ -67,7 +68,7 @@ const (
 func NewTriggerSubscriber(config *commoncap.RemoteTriggerConfig, capInfo commoncap.CapabilityInfo, capDonInfo commoncap.DON, localDonInfo commoncap.DON, dispatcher types.Dispatcher, aggregator types.Aggregator, lggr logger.Logger) *triggerSubscriber {
 	if aggregator == nil {
 		lggr.Warnw("no aggregator provided, using default MODE aggregator", "capabilityId", capInfo.ID)
-		aggregator = aggregation.NewDefaultModeAggregator(uint32(capDonInfo.F + 1))
+		aggregator = aggregation.NewDefaultModeAggregator(uint32(aggregation2.ByzantineQuorum(len(capDonInfo.Members), int(localDonInfo.F))))
```

Comment:
I'm afraid we can't set this here just yet or it will break the new LLO Streams Trigger. They only send F+1 copies. We first need to put extra info in the cap registry on whether a capability is OCR-based or not. Then based on that set the default. Likely also add the optimization to check each one and accept the first one with valid signatures. cc @krehermann